tests/integration/test_ethereum_tester.py

```diff
@@ -642,11 +642,32 @@ def test_eth_getBlockByNumber_finalized(
     ) -> None:
         super().test_eth_getBlockByNumber_finalized(w3, empty_block)
 
-    def test_eth_fee_history(self, w3: "Web3") -> None:
-        super().test_eth_fee_history(w3)
+    def test_eth_getBlockReceipts_hash(
```

Comment:
Since we don't have `eth_getBlockReceipts` implemented in eth-tester, we'll need to add these tests to the `not_implemented` section that's closer to the beginning of this file.

Reply:
Excuse me, may I ask if this is the change? It seems that even after the change here, it has not taken effect

Reply:
@kclowes /ping

Reply:
Thanks for the ping. Now that you are listing the tests as `not_implemented` above, you can delete them here and the tests should pass.